io/include/pcl/io/vlp_grabber.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2012 The MITRE Corporation
```

Comment:
Should this be rather 2015? Also, I think we need OpenPerception in the list of copyright holders.

```
Copyright (c) 2015-, Open Perception, Inc.
```